internal/key/key.go

```diff
@@ -122,15 +123,11 @@ func GetNodeInstanceProfile(machinePoolID string, clusterID string) string {
 	return fmt.Sprintf("nodes-%s-%s", machinePoolID, clusterID)
 }
 
-// IsCAPAVersion returns whether a given GS Release Version is based on the CAPI/CAPA projects
-// TODO: make this a >= comparison
-func IsCAPAVersion(version string) bool {
-	return strings.HasPrefix(version, "20.0.0")
-}
-
-// IsCAPZVersion returns whether a given GS Release Version is based on the CAPI/CAPZ projects
-func IsCAPZVersion(version string) bool {
-	return strings.HasPrefix(version, "20.0.0")
+// IsCAPIVersion returns whether a given GS Release Version uses the CAPI projects
+func IsCAPIVersion(version string) bool {
+	capiVersion, _ := semver.New(FirstCAPIRelease)
+	releaseVersion, _ := semver.New(version)
```

Comment:
stupid question but do we somehow check if the release has the proper format? could there be a scenario where someone inters the wrong semver and this would cause an error here?

Reply:
maybe `MustParse` could be better here?

Reply:
I added error handling. Now when using wrong formatted release, this error shows up
```
$ ./kubectl-gs-v1.45.0-dd192b5f87f39e5c13d71e24e204e66ca332dd72-linux-amd64 template cluster --context gs-godsmack --provider azure --release asd --organization giantswarm --description 'e2e test' --output cluster.yaml

Error: Parsing release: No Major.Minor.Patch elements found
```